onecodex/cli.py

```diff
@@ -264,6 +280,80 @@ def upload(ctx, files, max_threads, clean, no_interleave, prompt, validate,
         sys.exit(1)
 
 
+def update_sample_tags_and_metadata(sample_uuids, ctx, tag_array, metadata_opts):
```

Comment:
Probably makes sense to move these functions to some other internal module / helpers file.